schema/v1alpha1/thermal.proto

```
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+syntax = "proto3";
+
+package proto.v1alpha1;
```

Buf (1.55.1)

Files with package "proto.v1alpha1" must be within a directory "proto/v1alpha1" relative to root but were in directory "schema/v1alpha1".

(PACKAGE_DIRECTORY_MATCH)